### ISSUE TITLE
[BUG] Missing IPv6 argument 

### ISSUE THREAD

**Issue by evertramos** [evertramos/nginx-proxy-automation#265]
Need to add `ENABLE_IPV6=true`to docker-compose.yml file, as of

https://github.com/nginx-proxy/nginx-proxy#ipv6-support

**Comment by Zageron** [comment]
fyi, for people looking for solution
```yml
services:
  nginx-proxy-automation-web:
    image: nginx:${NGINX_IMAGE_VERSION:-stable-alpine}
    labels:
        com.github.jrcs.letsencrypt_nginx_proxy_companion.nginx_proxy: "true"
    container_name: ${NGINX_WEB_SEVICE_NAME:-nginx-proxy-automation-web}
    restart: always
    ports:
      - "${IPv4:-0.0.0.0}:${DOCKER_HTTP_:-80}:80"
      - "${IPv4:-0.0.0.0}:${DOCKER_HTTPS:-443}:443"
      - "${IPv6:-::0}:${DOCKER_HTTP_:-80}:80"
      - "${IPv6:-::0}:${DOCKER_HTTPS:-443}:443"
    environment:
      ENABLE_IPV6: "true"
      SSL_POLICY: ${SSL_POLICY:-Mozilla-Intermediate}
    volumes:
      - ${NGINX_FILES_PATH:-./data}/conf.d:/etc/nginx/conf.d
      - ${NGINX_FILES_PATH:-./data}/vhost.d:/etc/nginx/vhost.d
      - ${NGINX_FILES_PATH:-./data}/html:/usr/share/nginx/html
      - ${NGINX_FILES_PATH:-./data}/certs:/etc/nginx/certs:ro
      - ${NGINX_FILES_PATH:-./data}/htpasswd:/etc/nginx/htpasswd:ro
    logging:
      driver: ${NGINX_WEB_LOG_DRIVER:-json-file}
      options:
        max-size: ${NGINX_WEB_LOG_MAX_SIZE:-4m}
        max-file: ${NGINX_WEB_LOG_MAX_FILE:-10}
```

**Comment by evertramos** [comment]
Issued solved on v0.5.2
